hack/generate/samples/internal/ansible/memcached.go

```diff
@@ -126,128 +127,3 @@ func GenerateMemcachedAnsibleSample(samplesPath string) {
 	memcached.Prepare()
 	memcached.Run()
 }
-
```

Comment:
The consts were centralized in the constants.go file since now we have 2 samples.
1 for the testdata and another to generate the mock data to run the molecule tests.